README.md

```diff
@@ -81,39 +81,57 @@ structure.
 
 ## Development
 
-To make changes to the playbook theme, you will need to install the
-dependencies:
+### Prerequisites
 
-```
-npm install
-go get github.com/errata-ai/vale
+- Ruby 2.6.6 (Defined in `.ruby-version` for use with [rbenv](https://github.com/rbenv/rbenv))
+- Node.js 12.13.0 (Defined in `.node-version` for use with [nodenv](https://github.com/nodenv/nodenvv))
+- Go 1.14 (Defined in `go-version` for use with [goenv](https://github.com/syndbg/goenv))
```

Comment:
It looks like `goenv` is unable to find and install Go 1.14.

`goenv install 1.14` results in `go-build: definition not found: 1.14`.

`goenv install --list` provides a list that stops at `1.12beta1`.

I had already run `brew update && brew upgrade goenv`, everything up to date.

I have now installed Go 1.14 via Homebrew, and goenv still produces the following pearls of wisdom:
```
➜  playbook git:(jekyllise-playbook) goenv local
1.14
➜  playbook git:(jekyllise-playbook) goenv version
goenv: version `1.14' is not installed (set by /Users/cristinarotaru/projects/playbook/.go-version)
```

Should we even use it?

Reply:
Ah, I installed via Git, which is why this confused me initially. It seems there's an ongoing issue with installs via Homebrew (documented here: https://github.com/syndbg/goenv/issues/80). If you uninstall and install via git, it should work. It seems like it's being actively looked at, so should be fixed soon.